contracts/script/deploy/certverifier/README.md

```diff
@@ -0,0 +1,42 @@
+## EigenDA Cert Verfier Deployer
+
+This script can be used to deploy an EigenDACertVerifier contract with specified security thresholds and quorum numbers required
+
```

Comment:
let's specify that deployments can only be done on an Ethereum network and this isn't supported for L3 rollups